spec/models/shared_examples/user_subscription_sourceable.rb

```diff
@@ -23,9 +23,9 @@
   end
 
   describe "#create_user_subscription" do
-    it "returns a created UserSubcription with the correct attributes" do
-      user_subscription_fields = %w[author_id subsciber_id subscriber_email user_subscription_sourceable_id
-                                    user_susbcription_sourceable_type]
+    it "returns a created UserSubscription with the correct attributes" do
+      user_subscription_fields = %w[author_id subscribe_id subscriber_email user_subscription_sourceable_id
```

Comment:
```suggestion
      user_subscription_fields = %w[author_id subscriber_id subscriber_email user_subscription_sourceable_id
```

@jsoref was this not meant to be `subscriber_id` instead of `subscribe_id`?

Reply:
Hmm, the latter seems like a better guess.